pom.xml

```diff
@@ -102,6 +102,12 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
```

Comment:
I have looked through your code and I cannot find where this dependency is being used. Could you make sure it is useful for it to be added to the codebase?

Reply:
It's required for the selenium test , this link kind of says the problem i was getting beforehand https://github.com/bonigarcia/webdrivermanager/issues/432